### DISCUSSION TITLE
Purchased Domain Theft or Inaccesability

### DISCUSSION THREAD

**Discussion by Digida** [vercel/community#2976] [Help] [answered]
### Summary

I purchased a vercel domain but am failing to point it to my site. On closer investigation the nameservers are ns1.thinktpi.com and ns2.thinktpi.com instead of ns1.vercel-dns.com and ns2.vercel-dns.com. Was my purchased domain given out?

### Steps to Reproduce

I need my site working on www.meridiancenter.com, but i cant.

**Comment by amyegan** [comment, marked as answer]
Hi @Digida. It looks like www.meridiancenter.com was purchased via a third party registrar, so you need to manage DNS settings with them. It looks like you do have a different domain, www.merdiancenter.com, which has Vercel as the registrar. If you want to use that one for your website, be sure to add it in your project settings.

**Reply by Digida** [reply]
@amyegan I purchased it from vercel. 

![meridiancenter domain](https://github.com/vercel/community/assets/92722581/452fa442-5865-4751-b2f8-2df7219efba7)

Yet when I try to add it shows me the thinktpi.com nameserver

**Comment by Digida** [comment]
sorry i misspelt the name as merdincenter instead of meridian center when purchasing. Is there a way i can correct this?

**Reply by amyegan** [reply]
Unfortunately, domains cannot be modified (for example, to fix a misspelling). A new domain registration is needed to acquire the correct domain name.

If you do not want to keep the incorrect domain, then you may also wish to disable auto-renew on the incorrect domain so that it will expire and not be invoiced for next year.
